### PR TITLE
PDP - Implementación de punto de inicio

### PR DESCRIPTION
**Pull request by negro89**
### Requerimiento
https://proyectos.andes.gob.ar/browse/PDP-79

### Funcionalidad desarrollada 

1. Actualiza a version de plex 7.19.0
2. Nuevo punto de inicio del paciente
3. Reagrupamiento de elementos versión mobile

### UserStory llegó a completarse

- [X] Si
- [ ] No
- [ ] No corresponde

### Requiere actualizaciones en la base de datos

- [ ] Si
- [X] No

### Requiere actualizaciones en la API

- [ ] Si
- [X] No

### Requiere actualizaciones en andes-test-integracion

- [ ] Si
- [X] No

### CODE REVIEW

**Comment by liquid36** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [5153](http://condorpiedra.andes.gob.ar:85/5153/mochawesome.html)
CYPRESS RUN: [4105](https://dashboard.cypress.io/projects/xr7gft/runs/4105/specs)
TEST START: 2021-09-22T17:51:05.670Z
TOTAL: 383
SUCCESS: 365
FAIL: 17
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [5157](http://condorpiedra.andes.gob.ar:85/5157/mochawesome.html)
CYPRESS RUN: [4109](https://dashboard.cypress.io/projects/xr7gft/runs/4109/specs)
TEST START: 2021-09-23T12:37:03.547Z
TOTAL: 400
SUCCESS: 395
FAIL: 4
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [5160](http://condorpiedra.andes.gob.ar:85/5160/mochawesome.html)
CYPRESS RUN: [4112](https://dashboard.cypress.io/projects/xr7gft/runs/4112/specs)
TEST START: 2021-09-23T13:22:39.712Z
TOTAL: 400
SUCCESS: 395
FAIL: 4
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [5171](http://condorpiedra.andes.gob.ar:85/5171/mochawesome.html)
CYPRESS RUN: [4121](https://dashboard.cypress.io/projects/xr7gft/runs/4121/specs)
TEST START: 2021-09-24T19:21:41.378Z
TOTAL: 396
SUCCESS: 380
FAIL: 15
SKIPPED: 1

File: projects/portal/src/app/pages/mi-inicio/mi-inicio.component.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    constructor(
+        private router: Router,
+        private pacienteService: PacientePortalService,
+        private el: ElementRef
+    ) { }
+
+
+    ngOnInit() {
+        this.paciente$ = this.pacienteService.me();
+        this.cards = CARDS.filter(c => c.path !== 'mi-inicio' && c.path !== 'mis-datos-personales');
```
Comment (juuliotero) [inline projects/portal/src/app/pages/mi-inicio/mi-inicio.component.ts:29]: Se podría guardar en cada card con un flag si se muestra en inicio o no

**Review by juuliotero — CHANGES_REQUESTED** [review]

File: projects/portal/src/app/components/paciente-detalle.component.ts [hunk 2]
```diff
@@ -27,13 +27,28 @@ export class PacienteDetalleComponent implements OnInit {
...
+            id: 2,
+            nombre: 'Error en mis datos personales',
+            operacion: 'error:datosPersonales'
+        },
+        {
+            id: 3,
+            nombre: 'Otro error',
+            operacion: 'error:otro'
+        }
+    ];;
```
[the remark this answers is not in the document]
Comment (negro89) [reply projects/portal/src/app/components/paciente-detalle.component.ts]: corregidooo

**Comment by liquid36** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [5222](http://condorpiedra.andes.gob.ar:85/5222/mochawesome.html)
CYPRESS RUN: [4168](https://dashboard.cypress.io/projects/xr7gft/runs/4168/specs)
TEST START: 2021-10-01T13:52:43.863Z
TOTAL: 401
SUCCESS: 394
FAIL: 6
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [5225](http://condorpiedra.andes.gob.ar:85/5225/mochawesome.html)
CYPRESS RUN: [4171](https://dashboard.cypress.io/projects/xr7gft/runs/4171/specs)
TEST START: 2021-10-01T17:19:36.188Z
TOTAL: 401
SUCCESS: 394
FAIL: 6
SKIPPED: 1

**Review by JSantarelli — APPROVED** [review]

**Review by juuliotero — APPROVED** [review]

**Comment by liquid36** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [5239](http://condorpiedra.andes.gob.ar:85/5239/mochawesome.html)
CYPRESS RUN: [4185](https://dashboard.cypress.io/projects/xr7gft/runs/4185/specs)
TEST START: 2021-10-04T19:20:32.394Z
TOTAL: 401
SUCCESS: 396
FAIL: 4
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: nhuenchuman  
BUILD NUMBER: [5242](http://condorpiedra.andes.gob.ar:85/5242/mochawesome.html)
CYPRESS RUN: [4188](https://dashboard.cypress.io/projects/xr7gft/runs/4188/specs)
TEST START: 2021-10-05T11:36:33.382Z
TOTAL: 383
SUCCESS: 375
FAIL: 7
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: nhuenchuman  
BUILD NUMBER: [5246](http://condorpiedra.andes.gob.ar:85/5246/mochawesome.html)
CYPRESS RUN: [4192](https://dashboard.cypress.io/projects/xr7gft/runs/4192/specs)
TEST START: 2021-10-05T13:07:58.841Z
TOTAL: 401
SUCCESS: 393
FAIL: 7
SKIPPED: 1

**Review by palita1991 — APPROVED** [review]

**Comment by liquid36** [comment]
USUARIO: nhuenchuman  
BUILD NUMBER: [5248](http://condorpiedra.andes.gob.ar:85/5248/mochawesome.html)
CYPRESS RUN: [4194](https://dashboard.cypress.io/projects/xr7gft/runs/4194/specs)
TEST START: 2021-10-05T14:06:16.089Z
TOTAL: 401
SUCCESS: 400
FAIL: 0
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: mpalavecino  
BUILD NUMBER: [5249](http://condorpiedra.andes.gob.ar:85/5249/mochawesome.html)
CYPRESS RUN: [4195](https://dashboard.cypress.io/projects/xr7gft/runs/4195/specs)
TEST START: 2021-10-05T14:08:27.002Z
TOTAL: 5
SUCCESS: 4
FAIL: 1
SKIPPED: 0
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Pull request merged**